File: eMovieFinder/eMovieFinder.Database/Entities/Country.cs
using eMovieFinder.Model.Utilities;
using System.ComponentModel.DataAnnotations;

namespace eMovieFinder.Database.Entities
{
    public class Country : TimeStampObject
    {
        [Key]
        public int Id { get; set; }
        public string CountryName { get; set; }
    }
}

File: eMovieFinder-UI/emovie_finder_mobile/lib/src/models/entities/country.dart
import 'package:json_annotation/json_annotation.dart';

import 'package:emovie_finder_mobile/src/models/utilities/time_stamp_object.dart';

part 'country.g.dart';

@JsonSerializable()
class Country extends TimeStampObject {
  final int? id;
  final String? countryName;

  Country({
    super.creationDate,
    super.modifiedDate,
    super.createdById,
    super.modifiedById,
    super.formattedCreationDate,
    super.formattedModifiedDate,
    this.id,
    this.countryName
  });

  factory Country.fromJson(Map<String, dynamic> json) => _$CountryFromJson(json);
  Map<String, dynamic> toJson() => _$CountryToJson(this);
}